rascal2/dialogs/custom_file_editor.py

```diff
@@ -0,0 +1,99 @@
+"""Dialogs for editing custom files."""
+
+import logging
+from pathlib import Path
+
+from PyQt6 import Qsci, QtWidgets
+from RATapi.utils.enums import Languages
+from RATapi.wrappers import start_matlab
+
+
+def edit_file(filename: str, language: Languages, parent: QtWidgets.QWidget):
```

Comment:
i've got them here because they're just 'helper functions' for generating the dialog. 

do you mean you want them in the file presenter.py as static functions, or you want them to be methods of the presenter class? I'm a little against putting them in the presenter class because as it stands, the field table widgets are more or less independent of the presenter, and I'd be wiring them together just for the sake of calling a static function inside the presenter (or for using presenter.logger instead of getting the logger with `getLogger`)